### DISCUSSION TITLE
Error encountered in plot_model_fit

### DISCUSSION THREAD

**Discussion by saukang** [google/lightweight_mmm#53] [Q&A] [answered]
Hi team,

I have encountered this error while trying to execute `plot_model_fit`, any ideas and thoughts about it? I have been following the demo notebook step by step but this happened when I used my dataset. I have ensured that all inputs are in the right format and etc.

![Screenshot 2022-08-19 174601](https://user-images.githubusercontent.com/71245948/185593022-793472b1-e114-491d-b748-28e9f4da21a2.png)

Any help on this would be appreciated! 🙏 Thank you in advance!

Best regards,
Sau Kang.

**Comment by pabloduque0** [comment, marked as answer]
Hi @saukang ! Thanks for opening this one!

So without know much about some shapes of your data, my first guess would be that you have fit you `target_scaler` on a target that has shape `(48, 1)` instead of `(48,)`. Could you confirm if that is the case and if that fixes it?

Otherwise I will need a bit more information about the shapes of your media data, target and the values within your scaler.

Best,

**Reply by saukang** [reply]
Hey @pabloduque0,

Thanks for the speedy response and you were right! There is no problem after the target array was flattened! Apologize for my rookie error, should have picked up the hint from the error message suggesting it was a broadcasting error.

Cheer! Thanks for the lightweight mmm tool!

Best,
Sau Kang.

**Reply by pabloduque0** [reply]
no worries at all! Happy to hear it was solved! :)
